cloudamqp/lavinmq
# UI updates

### WHAT is this pull request doing?
Major changes:
- New color scheme and profile (buttons, borders, sizes etc)
- New typeface
- New layout

New color scheme:
Dark color scheme as a base now. Plan is to add light mode later.

New typeface:
Added Inter (https://rsms.me/inter/).

New layout:
Biggest change is that the header menu and sidebar is now fixed.

### HOW can this pull request be tested?
Run LavinMQ, check out new UI.

Feel free to test so that the functionality is still intact. Preferably focus on the layout with fixed menus.
